src/coverlet.collector/DataCollection/CoverageManager.cs

```diff
@@ -23,28 +23,28 @@ internal class CoverageManager
         public IReporter[] Reporters { get; }
 
         public CoverageManager(CoverletSettings settings, TestPlatformEqtTrace eqtTrace, TestPlatformLogger logger, ICoverageWrapper coverageWrapper,
-                               IInstrumentationHelper instrumentationHelper, IFileSystem fileSystem, ISourceRootTranslator sourceRootTranslator)
+            IInstrumentationHelper instrumentationHelper, IFileSystem fileSystem, ISourceRootTranslator sourceRootTranslator)
```

Comment:
Sorry I wasn't clear, I don't want any update on this file, revert completely to master using command like `git checkout master src/coverlet.collector/DataCollection/CoverageManager.cs `

Reply:
Done